docs/source/blocks/extensions.md

```diff
@@ -160,6 +160,17 @@ The `schemaEnhancer` is a generic extension mechanism provided by
 it for the `variation` extension.
 ```
 
+Volto provides a helper to combine multiple schemaEnhancers into a single
```

Comment:
It is confusing to have a mix of "schemaEnhancers" and "schema enhancers". In narrative text, I think the latter is preferred, whereas in code the former should be an inline literal.

```suggestion
Volto provides a helper to combine multiple schema enhancers into a single
```